### PR TITLE
[ORCA-3895] Add docs for the Global Orchestration resource

### PR DESCRIPTION
**Pull request by c-kaieong**
WISOTT

### CODE REVIEW

File: website/docs/r/event_orchestration.html.markdown [hunk 1]
```diff
@@ -3,14 +3,14 @@ layout: "pagerduty"
 page_title: "PagerDuty: pagerduty_event_orchestration"
 sidebar_current: "docs-pagerduty-resource-event-orchestration"
 description: |-
-  Creates and manages a Global Event Orchestration in PagerDuty.
+  Creates and manages an Event Orchestration in PagerDuty.
```
Comment (c-kaieong) [inline website/docs/r/event_orchestration.html.markdown:6]: In the new Orchestrations v2 world, `Global Event Orchestration` refers to a different concept / resource. In this PR, we are renaming this specific metadata resource as `Event Orchestration` to be more in-line with our other public facing docs.

File: website/docs/r/event_orchestration.html.markdown [hunk 2]
```diff
@@ -3,14 +3,14 @@ layout: "pagerduty"
...
 ---
 
 # pagerduty_event_orchestration
 
-[Global Event Orchestrations](https://support.pagerduty.com/docs/event-orchestration#global-orchestrations) allow you define a set of Event Rules, so that when you ingest events using the Orchestration's Routing Key your events will be routed to the correct Service, based on the event's content.
+[Event Orchestrations](https://support.pagerduty.com/docs/event-orchestration) allow you define a set of Event Rules, so that when you ingest events using the Orchestration's Routing Key your events will be routed to the correct Global and/or Service Orchestration, based on the event's content.
```
Comment (c-kaieong) [inline website/docs/r/event_orchestration.html.markdown:11]: Hopefully by the time this PR gets merged, the [v1](https://support.pagerduty.com/docs/event-orchestration) KB article will be replaced with the [v2](https://support.pagerduty.com/docs/event-orchestration-v2-early-access) article

File: website/docs/r/event_orchestration_global.html.markdown [hunk 3]
```diff
@@ -0,0 +1,159 @@
+---
```
Comment (c-kaieong) [inline website/docs/r/event_orchestration_global.html.markdown:1]: This whole markdown file is heavily based off of `event_ochestration_service.html.markdown`

File: website/docs/r/event_orchestration_global.html.markdown [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+layout: "pagerduty"
+page_title: "PagerDuty: pagerduty_event_orchestration_global"
+sidebar_current: "docs-pagerduty-resource-event-orchestration-global"
+description: |-
+  Creates and manages a Global Orchestration for an Event Orchestration.
+---
+
+# pagerduty_event_orchestration_global
+
+A [Global Orchestration](https://support.pagerduty.com/docs/event-orchestration#global-orchestrations) allows you to create a set of Event Rules. The Global Orchestration evaluates Events sent to it against each of its rules, beginning with the rules in the "start" set. When a matching rule is found, it can modify and enhance the event and can route the event to another set of rules within this Global Orchestration for further processing.
```
Comment (c-kaieong) [inline website/docs/r/event_orchestration_global.html.markdown:11]: This KB article URL still points to v1, which contains inaccurate information pertaining to what a Global Orchestration is in the new v2 world.

Once the v1 KB article gets replaced by the v2 article, this URL should hopefully point to the correct information.

File: website/docs/r/event_orchestration.html.markdown [hunk 5]
```diff
@@ -3,14 +3,14 @@ layout: "pagerduty"
...
 
-## Example of configuring a Global Event Orchestration
+## Example of configuring a Event Orchestration
```
Comment (alenapan) [inline website/docs/r/event_orchestration.html.markdown:13]: a -> an
```suggestion
## Example of configuring an Event Orchestration
```

File: website/docs/r/event_orchestration_global.html.markdown [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+* `label` - (Optional) A description of this rule's purpose.
+* `condition` - (Optional) Each of these conditions is evaluated to check if an event matches this rule. The rule is considered a match if any of these conditions match. If none are provided, the event will `always` match against the rule.
+* `actions` - (Required) Actions that will be taken to change the resulting alert and incident, when an event matches this rule.
+* `disabled` - (Optional) Indicates whether the rule is disabled and would therefore not be evaluated.
+
+### Condition (`condition`) supports the following:
+* `expression`- (Required) A [PCL condition](https://developer.pagerduty.com/docs/ZG9jOjM1NTE0MDc0-pcl-overview) string.
+
+### Actions (`actions`) supports the following:
+* `route_to` - (Optional) The ID of a Set from this Global Orchestration whose rules you also want to use with event that match this rule.
```
Comment (alenapan) [inline website/docs/r/event_orchestration_global.html.markdown:110]: I know the same wording is used in Service and Unrouted docs but should this say "**events** that match" (plural)?
```suggestion
* `route_to` - (Optional) The ID of a Set from this Global Orchestration whose rules you also want to use with events that match this rule.
```

File: website/docs/r/event_orchestration_global.html.markdown [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+* `drop_event` - (Optional) When true, this event will be dropped. Dropped events will not trigger or resolve an alert or an incident. Dropped events will not be evaluated against router rules.
+* `suppress` - (Optional) Set whether the resulting alert is suppressed. Suppressed alerts will not trigger an incident.
+* `suspend` - (Optional) The number of seconds to suspend the resulting alert before triggering. This effectively pauses incident notifications. If a `resolve` event arrives before the alert triggers then PagerDuty won't create an incident for this the resulting alert.
```
Comment (alenapan) [inline website/docs/r/event_orchestration_global.html.markdown:113]: Could we change the wording to "this alert" here and also in website/docs/r/event_orchestration_service.html.markdown?
```suggestion
* `suspend` - (Optional) The number of seconds to suspend the resulting alert before triggering. This effectively pauses incident notifications. If a `resolve` event arrives before the alert triggers then PagerDuty won't create an incident for this alert.
```

**Review by alenapan — COMMENTED** [review]
Added a couple of comments about typos but most of them were here before these changes, otherwise LGTM 👍

**Comment by alenapan** [comment]
Oh I also changed the base branch to ORCA-3893-global-orchestrations where the we actually added support for Global orchestrations, so we can merge (and revert if necessary) all GO related changes at once.

**Pull request merged**